### PR TITLE
New EasyConfig for ZeroMQ 4.1.4 with the foss-2015a toolchain

### PR DESCRIPTION
**Pull request by valtandor**
needs ~~#2517~~ ~~#2516~~

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
lgtm

File: easybuild/easyconfigs/z/ZeroMQ/ZeroMQ-4.1.4-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# --with-pgm will use shipped OpenPGM (in foreign subdir)
+configopts = '--with-pic --with-system-pgm '
+configopts += 'OpenPGM_CFLAGS="-I$EBROOTOPENPGM/include/pgm-${EBVERSIONOPENPGM%.*}" '
+configopts += 'OpenPGM_LIBS="-L$EBROOTOPENPGM/lib -lpgm -lrt -lpthread -lm" '
+
+dependencies = [
+    ('OpenPGM', '5.2.122'),
+    ('util-linux', '2.27.1'),
+    ('libsodium', '1.0.6'),
```
Comment (boegel) [inline easybuild/easyconfigs/z/ZeroMQ/ZeroMQ-4.1.4-foss-2015a.eb]: #2516 provides libsodium 1.0.*8*, not 1.0.*6*

**Comment by valtandor** [comment]
@boegel: Fixed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cba719aec4e0241bcb99 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**